app/controllers/declaration/SealController.scala

```diff
@@ -108,15 +107,9 @@ class SealController @Inject()(
     }
 
   private def updateCache(sessionId: String, formData: Seq[Seal])(implicit req: JourneyRequest[_]): Future[Unit] =
-    for {
-      _ <- getAndUpdateExportCacheModel(
-        sessionId,
-        model =>
-          exportsCacheService
-            .update(sessionId, model.copy(seals = formData))
-      )
-      _ <- customsCacheService.cache[Seq[Seal]](cacheId, formId, formData)
-    } yield Unit
+    getAndUpdateExportCacheModel(sessionId, model => {
+      exportsCacheService.update(sessionId, model.copy(seals = formData))
+    }).map(_ => ())
```

Comment:
Do we need map this to Unit? is it necessary? It doesn't matter what this function return, because in code where we use it we just use `map( _ => ...)`